modules/installing-ove-console-final.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-booting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-console-final_{context}"]
+= Completing cluster configuration and initiating the installation
+
+Before you can finally initiate the cluster installation, you must verify host details, configure cluster networking details, and download the default cluster credentials.
+
+.Prerequisites
+
+* You have booted all of the hosts that will comprise your cluster and configured them with the correct rendezvous node IP address.
++
+[IMPORTANT]
+====
+You can add as many hosts as you want to your cluster.
+However, at this stage, you must at least have enough available hosts to match the value you specified in the *Number of control plane nodes* field of the installation console's *Cluster details* page.
+====
+
+.Procedure
+
+. On the installation console hosted by the rendezvous node, configure the hosts in the *Host discovery* page:
+
+.. Verify that every machine you booted from the ISO image appears in the *Host Inventory* section and has a *Status* value of *Ready*.
+
+.. For each host, click the expand icon and verify that all of the specification fields are correct.
+
+.. Optional: For each host, configure the role by selecting an option from the dropdown menu of the *Role* column.
+The default value for every host except the rendezvous node is `Auto-assign`.
+
+.. Click *Next* to continue.
+
+. On the *Storage* page, click the expand icon for each host and verify that all of the specification fields are correct.
+
+. Click *Next* to continue.
+
+. Configure the cluster networking on the *Networking* page:
+
+.. Optional: Select a machine network from the dropdown menu of the *Machine network* field.
```

Comment:
This section still looks a bit confusing to me. In the `Networking` page by default the `Cluster-managed Networking` is select: 
![image](https://github.com/user-attachments/assets/8d619fb4-6f15-4b16-b14e-214e6a419b6c)

With this option, the user is required to enter the `API` and `Ingress` vips,as well as the stack type.

Instead, by selecting `User-Managed Networking option` does not have to specify anything:
![image](https://github.com/user-attachments/assets/f9467284-248c-4ce8-a318-8493ecbc7c5a)

Note that the `Use advanced networking` and SSH public key fields are always (optionally) available in both the cases.

Reply:
I did not know this was how the UI worked, let me try to rewrite this section to account for these two options.